src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.ts

```diff
@@ -106,7 +125,126 @@ export class EditDatumGroepComponent extends EditComponent {
         this.fataleDatumField.formControl.setValue(this.fataleDatum);
         this.showStreefError = false;
         this.showFataleError = false;
-
         this.editing = false;
     }
+
+    private maakOpschortDuurField(): InputFormField {
+        this.opschortDuurField = new InputFormFieldBuilder()
```

Comment:
Ik zou deze methode alleen het InputFormFieldBuilder() object laten maken en meteen retourneren en het toekennen aan this.opschortDuurField hier buiten laten.

Reply:
Die heb ik nodig voor de afhankelijkheden tussen de drie velden.